core/src/replay_stage.rs

```diff
@@ -769,9 +771,18 @@ impl ReplayStage {
         let verify_result = entries.verify(last_entry);
         let verify_entries_elapsed = now.elapsed().as_micros();
         bank_progress.stats.entry_verification_elapsed += verify_entries_elapsed as u64;
-        if !verify_result {
-            info!(
-                "entry verification failed, slot: {}, entry len: {}, tick_height: {}, last entry: {}, last_blockhash: {}, shred_index: {}",
+        let hashes_per_tick = bank.hashes_per_tick().unwrap_or(0);
```

Comment:
I also need a check to make sure that the tick entries in `entries` won't exceed `bank.max_tick_height`

Reply:
yeah, would be good to cut that off early

banks that run over can't earn a vote, because we check 'em as we go, but I don't think we have any protection in replay stage against a bank that never dies

might also be nice to mark the slot as dead in blocktree so that window and retransmit don't waste any time on it

Reply:
Cool, I think this PR covers that now!